docs/TIPS.md

```diff
@@ -0,0 +1,251 @@
+# Development tips
+
+## General
+
+### Exceptions and stack unwinding
+
+Having a custom unwinder, Pyston has a few special constraints - see [exception safety](EXCEPTION-SAFETY.md) and [unwinding](UNWINDING.md).
+
+### Make shortcuts
+
+The main Make targets you will use will be
+- `make`: to compile in debug mode and generate the `pyston_dbg` executable
+- `make pyston_release`: to compile in release mode and generate the `pyston_release` executable
+- `make check`: run the tests
```

Comment:
I'd switch `quick_check` and `check` - `quick_check` here, `check` down in the larger list.